### ISSUE TITLE
modals with timeout?

### ISSUE THREAD

**Issue by ghost** [kylepaulsen/NanoModal#9]
I would like a dialog which returns with some predefined value after a short timeout if the user didn't click anything - is that doable?

**Comment by kylepaulsen** [comment]
Do you mean something like this?
https://jsfiddle.net/psfz2gxg/

**Comment by ghost** [comment]
yes, this is exactly what I had on my mind. seems so easy once you see the solution..

**Comment by ghost** [comment]
of course it is a good idea to remove the timer if the user previously hit an option
